preview-src/page-templates/full.adoc

```diff
@@ -0,0 +1,74 @@
+= {company} Astra
+:page-layout: full
+
+[.max-w-[650px]]
+DataStax Astra is a cutting-edge cloud-native database platform designed to empower developers.
+Harnessing the strengths of vector databases, serverless computing, and real-time streaming, Astra propels application development into the next generation.
+From AI-driven capabilities to robust data stream management, Astra simplifies complex tasks, offering unparalleled performance and scalability.
+
+[.[&>h2]:!hidden]
+== {empty}
+
+++++
+<div class="flex rounded-xl bg-[var(--ds-background-level1)] p-2 lg:p-3 gap-3 -mx-2 lg:-mx-3">
+<div class="flex flex-col lg:basis-1/2">
+++++
+
+[discrete.!m-0]
+== [.rounded-xl.border.p-1.flex.w-max.mb-1]#icon:/img/astra-database.svg[]# Vector
```

Comment:
These icons aren't showing up. Is it possible to add SVG icons that use CSS variables?

Reply:
So the issue is that my svg macros don't work in the preview. It's because the macros use the content catalog, but in the preview that doesn't really seem to be available unfortunately

Reply:
Oh interesting. So they're basically hybrid Asciidoctor/Antora extensions? I guess that makes sense since we want them to be able to use Antora's resource ID system to identify files, e.g. `ROOT:icons/astra-database.svg[]`. I assume that's the reason for the content catalog dependency?